app/src/main/java/org/wikipedia/extensions/View.kt

```diff
@@ -0,0 +1,11 @@
+package org.wikipedia.extensions
+
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.lifecycleScope
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+
+fun View.coroutineScope(): CoroutineScope {
```

Comment:
Would it be better if we provide an optional parameter to send a `Dispatchers`? 

`View.coroutineScope(coroutineContext: CoroutineContext = Dispatchers.Main)`

Reply:
sure!